tests/test_evaluation.py

```diff
@@ -5,23 +5,24 @@
 from qiskit import QuantumCircuit
 
 from mqt.bench import get_benchmark
-from mqt.predictor import Result, rl
+from mqt.bench.devices import get_available_device_names, get_available_devices
+from mqt.predictor import Result
 from mqt.predictor.evaluation import create_qiskit_result, create_tket_result, evaluate_sample_circuit
 
 
 def test_evaluate_sample_circuit() -> None:
     qc = get_benchmark("ghz", 1, 3)
     filename = "test_3.qasm"
     qc.qasm(filename=filename)
-    res = evaluate_sample_circuit(filename)
+    res = evaluate_sample_circuit(filename, get_available_devices())
     expected_keys = []
     for compilation_setup in ["qiskit", "tket", "mqt-predictor_expected_fidelity", "mqt-predictor_critical_depth"]:
         for key in ["time", "expected_fidelity", "critical_depth"]:
             if "mqt-predictor" in compilation_setup:
                 expected_keys.append(compilation_setup + "_" + key)
             else:
                 expected_keys.extend(
-                    [compilation_setup + "_" + device["name"] + "_" + key for device in rl.helper.get_devices()]
+                    [compilation_setup + "_" + device_name + "_" + key for device_name in get_available_device_names()]
```

Comment:
Could we store the available_devices from line 17 and iterate over those instead of calling `get_available_device_names`. By that, we would get rid of one function call and import.

Reply:
Now that we have removed the `devices` argument in `evaluate_sample_circuit()` we still need tha names.
Dont worry though, the name method is really not expensive.